vars/checkPatch.groovy

```diff
@@ -1,13 +1,33 @@
 // vars/checkPatch.groovy
 
+/**
+ * checkPatch.groovy
+ *
+ * checkPatch variable
+ */
+
 import com.intel.checkoutScm
 
-def call(Map config) {
+/**
+ * Method to check the status of a patch and notify GitHub of the results
+ * 
+ * @param config Map of parameters passed.
+ *
+ * config['jenkins_review'] is the optional revision number to use.
+ *
+ */
+def call(Map config = [:]) {
+    if (env.DAOS_JENKINS_NOTIFY_STATUS == null) {
+      println "Jenkins not configured to notify github of builds."
+      stepResult name: env.STAGE_NAME, context: "pre-build",
+                 result: "SUCCESS"
```

Comment:
If we don't want to notify GitHub why even call `stepResult` here?

Reply:
Not sure.
In the Jenkinsfile I am also calling stepResult.
Maybe we need some logic in stepResult also.
There should be ways like environment variables to tell us how the run was triggered.

Reply:
This seems unresolved at this point.